### PR TITLE
Weather app by KatSjolin

### PR DESCRIPTION
**Pull request by ghost**
## Netlify link
https://rococo-sunburst-79526b.netlify.app/

### CODE REVIEW

**Comment by pernillasterner** [comment]
### Feedback 👩‍💻
👌 I really appreciate that you've kept the code clean, and it looks good on all devices, so nicely done! 

🌱 One thing to consider for future projects is to add more comments. They will help you easily understand the code better, especially if you need to revisit it in a month or so. Having some guidance in the form of comments will make it easier to understand and follow the code.

👏 I can see that you have a passion for styling, and you've dedicated time to make it as close to the design as possible.

File: .vscode/settings.json
Comment (AntonellaMorittu) [inline .vscode/settings.json:1]: please delete this folder 🧹

File: index.html
Comment (AntonellaMorittu) [inline index.html:1]: please format this code 🪄

**Review by AntonellaMorittu — APPROVED** [review]
Hi Katarina, very good job this week! Your code is well structured and the app is working well, you completed the basic requirements to pass this assignment 🥳 
I left some comments about formatting and cleaning the code, keep up the good work! 💯

**Pull request closed without merging**